assets/js/base/components/payment-methods/no-payment-methods/style.scss

```diff
@@ -0,0 +1,24 @@
+
+.wc-block-checkout__no-payment-methods {
+	display: block;
+	margin-bottom: $gap !important;
```

Comment:
All comments I did above about removing `display: block` and exchanging `!important` for a more specific selector apply here too.